libstorage/SQLBasicAccess.h

```diff
@@ -75,12 +75,27 @@ class SQLBasicAccess
 
     int CommitDo(int64_t _num, const std::vector<TableData::Ptr>& _datas, std::string& _errorMsg);
 
+    std::string getFieldType(std::string const& _tableName);
+    bool inline isBlobType(std::string const& _tableName)
+    {
+        auto fieldType = getFieldType(_tableName);
+        if (fieldType == c_mediumBlobType || fieldType == c_longBlobType)
+        {
+            return true;
+        }
+        return false;
+    }
+
 public:
     virtual void ExecuteSql(const std::string& _sql);
     void setConnPool(std::shared_ptr<SQLConnectionPool>& _connPool);
 
 private:
     std::shared_ptr<SQLConnectionPool> m_connPool;
+    std::string const c_mediumBlobType = "mediumblob";

```

Comment:
Why not just use enum?

Reply:
Enum has been used instead of string comparison.